TCAT/Cells/BusStopCell.swift

```diff
@@ -38,7 +38,7 @@ class BusStopCell: UITableViewCell {
         imageView?.frame = CGRect(x: 10.0, y: 5.0, width: imageWidth, height: imageHeight)
         imageView?.contentMode = .scaleAspectFit
         imageView?.center.y = bounds.height / 2.0
-        imageView?.image = reuseIdentifier == "currentLocation" ? #imageLiteral(resourceName: "location") : #imageLiteral(resourceName: "pin")
+        imageView?.image = reuseIdentifier == Key.Cells.currentLocationIdentifier ? #imageLiteral(resourceName: "location") : #imageLiteral(resourceName: "pin")
```

Comment:
i think it's good you're replacing the string with a var  name. however, i think the cell identifier should not be in a global file but rather be a `let` variable at the top of the cell class b/c the only place you're going to use the identifier is in the cell.

Reply:
It's used in a few places